dev/release/02-source.sh

```diff
@@ -56,12 +56,43 @@ tarball=$tag.tar.gz
 # archive (identical hashes) using the scm tag
 git archive $release_hash --prefix $tag/ -o $tarball
 
+# download apache rat
+curl -s https://repo1.maven.org/maven2/org/apache/rat/apache-rat/0.12/apache-rat-0.12.jar > apache-rat-0.12.jar
```

Comment:
Probably the best choice is to just add this command to the travis shell script.